### PR TITLE
Real world data example for Silhouette Visualizer - "Receivers" dataset

### PR DESCRIPTION
**Pull request by mike-curry00**
This PR fixes part of #516 and is a redo of #878. I submitted a PR on the **yellowbrick-datasets** project, too (https://github.com/DistrictDataLabs/yellowbrick-datasets/pull/7).

Quick recap... The following Silhouette Plot is generated by using statistics on all eligible receivers in the 2018 NFL regular season:

image

I was able to get an email from Sports Reference, LLC to confirm that we are allowed to use their data, so there should be no issues from a permissions standpoint.

**To-Do once the dataset is available in the S3 bucket:**
- [x] Edit yellowbrick/datasets/manifest.json to include the correct _url_ for "receivers"
- [x] Update docs/api/cluster/silhouette.rst

Question for the maintainers - should this example (or the Silhouette plot image) be used in any other sections of the project?

### CODE REVIEW

**Comment by lwgray** [comment]
@mike-curry00 great Job!  I am looping in @bbengfort to help you upload the dataset to S3

**Comment by bbengfort** [comment]
Currently blocked by https://github.com/DistrictDataLabs/yellowbrick-datasets/pull/7

File: yellowbrick/datasets/manifest.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
     "url": "https://s3.amazonaws.com/ddl-data-lake/yellowbrick/v1.0/occupancy.zip",
     "signature": "0b390387584586a05f45c7da610fdaaf8922c5954834f323ae349137394e6253"
   },
+  "receivers": {
+    "url": "receivers.zip",
+    "signature": "a5dbe22d267e9aa12bd84be325c417c27882d156d4410a718262aad6e235ab9b"
+  },
```
Comment (bbengfort) [inline yellowbrick/datasets/manifest.json]: This should now be:

```
"nfl": {
    "url": "https://s3.amazonaws.com/ddl-data-lake/yellowbrick/v1.0/nfl.zip",
    "signature": "e2d0665ed4fd7bea866ba6d2f0035fce89e6143da3c12dcd870e67319451a382"
  }
```

I've updated yellowbrick-datasets with the new manifest if you just want to copy it over.

**Comment by bbengfort** [comment]
@mike-curry00 the manifest from [yellowbrick-datasets PR #8](https://github.com/DistrictDataLabs/yellowbrick-datasets/pull/8) should be updated to:

```
  "nfl": {
    "url": "https://s3.amazonaws.com/ddl-data-lake/yellowbrick/v1.0/nfl.zip",
    "signature": "4989c66818ea18217ee0fe3a59932b963bd65869928c14075a5c50366cb81e1f"
  }
```

Hopefully, this will help resolve the README encoding issues!

**Comment by mike-curry00** [comment]
I'm having trouble with the Javascript LGTM analysis check (build: "No Javascript or Typescript code found"). Unless this is supposed to look through the jupyter notebooks, I don't think there's any JS in here? @Kautumn06, @bbengfort, are either of you familiar with this type of error?

**Comment by bbengfort** [comment]
@mike-curry00 don't worry too much about the LGTM JavaScript analysis, the only checks that block merging are the Travis CI and AppVeyor tests. I'll take a look and see why the analysis failed, but I think that only involves our documentation.

**Comment by bbengfort** [comment]
@mike-curry00 also, I'm going to merge the latest from develop into this PR - please make sure you do a pull before you make any more changes!

**Comment by rebeccabilbro** [comment]
FYI @mike-curry00, since it looks like this one is almost wrapped &mdash; if you're game to work on some follow-on PRs after this one is done, there are a couple that relate to switching from `make_clusters` to a real-world clustering dataset:
- #311 
- #437 
- #515 

Would love to have you contribute again!

**Comment by mike-curry00** [comment]
Certainly, @rebeccabilbro. I'd be happy to help!

**Pull request merged**